Update 04-writing.Rmd (#19)
fhdsl/Overleaf_Tutorial: 04-writing.Rmd

We will refer to these tags with brackets such as `\usepackage{}` as **commands** from now on (as this is what the are generally referred to) and they cause a change to either the text within the brackets or the overall document.

> The `\usepackage{}` command installs packages, which are bits of  code that help you do additional things with your documents. Packages need to be installed before commands from those packages can be called. Commands from these packages will be utilized later in the template. **It is recommended that you leave this code as is, and only modify the rest of the template until you learn more.** 

Reviewer: ```suggestion
The `\usepackage{}` command installs packages, which are collections of  code that help you do additional things with your documents. Packages need to be installed before commands from those packages can be used. Commands from these packages will be utilized later in the template. **It is recommended that you leave this code as is, and only modify the rest of the template until you learn more.** 
```

If you do not close a set of brackets you will get an error, so be careful about this.
</div>

As we scroll down the template, we next see the `\title` command that we previously worked with when we modified the text within the brackets to change the title. 

```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Recap of the title command"}
ottrpal::include_slide("https://docs.google.com/presentation/d/1UgGtVn7RsqdQ4pJxDk_dueSyREHcH-uWTNAT27E2mG8/edit#slide=id.g1d07d88c199_0_12")
```

> Then we see the `\author` command, which adds authors to the paper. These will be formatted in the way that is shown on the template. When you see `\\` two backslashes, this indicates that the line is finished and a new one is to be made. For tables or formatting like the authors, it should work well, but it is best to  **[avoid](https://tex.stackexchange.com/questions/225893/what-does-double-backslash-in-latex-mean)** using this for line breaks within the paragraphs that you might include in the paper. 

Reviewer: ```suggestion 
Then we see the `\author` command, which adds authors to the paper. These will be formatted in the way that is shown on the template. When you see `\\` two backslashes, this indicates that the line is finished and a new one is to be made. For tables or formatting like the authors, it should work well, but it is best to **[avoid](https://tex.stackexchange.com/questions/225893/what-does-double-backslash-in-latex-mean)** using this for line breaks within the paragraphs that you might include in the paper. 
```

### Abstract

The abstract section can be distinguished using the `begin{}` and `end{}` functions just like we used for the body of the document. These two commands will also be used later to indicate that a specific part of the document has started or ended.

### Dummy text

> You may notice `\lipsum[]` is used to create random chunks of text. The number within the brackets indicates what specific dummy paragraph to use; the maximum value allowed in the brackets is 150.

Reviewer: ```suggestion
You may notice `\lipsum[]` is used to create random chunks of text. The number within the brackets indicates what specific dummy paragraph to use. There are 150 possible, so the maximum value allowed in the brackets is 150.
```
